topics/kubernetes/solutions/killing_containers.md

```diff
@@ -9,4 +9,4 @@
 
 ## After you complete the exercise
 
-* Why did the "RESTARTS" count raised? - `because we killed the process and Kubernetes identified the container isn't running properly so it performed restart to the Pod`
+* Why did the "RESTARTS" count raised? - `because we killed the process and Kubernetes identified the container isn't running properly so it performed a restart to the Pod`
```

Comment:
maybe replace it with:
```suggestion
* Why did the "RESTARTS" count raised? - `Kubernetes restarted the Pod because we killed the process and the container was not running properly.`
```